proposals/841-release-process/README.md

```diff
@@ -0,0 +1,117 @@
+
+## Motivation
+
+Current challenges based on Kubeflow release 1.10:
+* Challenge to get component’s release progress during release meetings.
+    - Blockers or challenges on the Release are identified later.
+    -  Incomplete/Inaccurate or null information on some components about progress/blockers.
+* The Release Manager role challenges:
+- Only one person is responsible for the Release and decisions.
+- Time dedication and commitment are high and for an extended period (6 months), which makes it hard to find candidates for Release.
+- The diverse skills required for the role make it even harder for candidates to join.
+* Release Documentation
+- Working groups need help on building Technical Documentation for Releases.
+- Working on technical documentation during the Release, before the feature freeze, can expedite the release process.
+* Feature Freeze vs Building Release Candidate
+- Teams required more time to finish the Release and later cherry-pick and work on building the Release.
+* Release timeframe 
+- The Release is now 27 weeks. There are 10 weeks between feature freeze and Release. 
+- Release dates and phases are hard to predict in the current process.
+
+### Goals
+
+* Cut down the release handbook by 70 %
+* Replace the the manual bureaucratic process and overhead and human hours needed by an an automation first approach
+* Move away from the Waterfall model to an agile model that relies on automated tests instead of human labor.
+* Reduce time to market of Kubeflow Release, by being able to release at any time
+* Reduce meeting time and structure to give back time to the Release Team.
+* Focus on contributions from technical documents and source code by reducing meetings and promoting asynchronous communication.
+* Promote clarity on dates and phases so distributions, working groups, and the whole community can plan accordingly.
+* Improve release cycle by moving technical decisions affecting the release to the Release Management team including Kubeflow platform
+
+### Non-Goals
+
+<!--
+What is out of scope for this KEP? Listing non-goals helps to focus discussion
+and make progress.
+-->
+
+## Proposal
+
+### Release Timelines
+KubeFlow Release x.x is moved to Quarterly Release = 12 weeks
+
+![alt text](release.png)
+
+* Notes: Release dates will not be changed unless critical changes are needed.
+
+* KubeFlow Release minor versions x.x.xx
+** Release can be 60-90 days.
+
+### Release Meetings and Communication
+* Create a new Release Management team on Slack for asynchronous communication within the team
+* Bi-weekly meetings during Software Development (Week 1-Week 8). Weekly meetings between week 9-12 as needed.
+* Meetings will focus on roadmap status, including blockers, feature discussions, the overall status of the release, and the help needed.
+
+### Release Documentation
+
+* Build a team of technical writers to be embedded in each working group for the release time frame. 
```

Comment:
I think this is fine but we should also try to generate release notes from git commits using semantic commits and GenAI to summarize it.